src/Microsoft.AspNetCore.Razor.Runtime/TagHelpers/TagHelperAttributeList.cs

```diff
@@ -25,6 +25,20 @@ public TagHelperAttributeList()
         /// </summary>
         /// <param name="attributes">The collection to wrap.</param>
         public TagHelperAttributeList(IEnumerable<TagHelperAttribute> attributes)
+            : base (new List<TagHelperAttribute>(attributes))
```

Comment:
Where is this copy actually required? I'm hoping this constructor could be `internal` and just `// Internal for testing`.